miraclelinux/meta-debian-extended
# add tiff

## How to test

tiff tests are assumed to be run in the source tree.
So, I built the source code on the target, and run the tests there as following.

* Add tools-sdk to EXTRA_IMAGE_EFATURES in local.conf , and run `bitbake core-image-minimal`.
* Run `bitbake giflib -c patch` on build host to get patched source.
* Run the image with `runqemu qemuarm64 nographic`.
* Take tiff source code directory (build/tmp-glibc/work/aarch64-emlinux-linux/tiff/4.1.0+git191117-r0/tiff-4.1.0+git191117) to the running image using scp.
* Build the source code, and run tests on the running image.

```
$ cd giflib-5.1.4
$ ./autogen.sh
$ ./configure
$ make
$ make check
```
